src/auth.rs

```diff
@@ -17,6 +21,53 @@ pub fn parse_auth(auth: String, host: &str) -> io::Result<(String, Option<String
     }
 }
 
+pub fn read_netrc() -> Option<String> {
+    let mut netrc_buf = String::new();
+    if let Some(mut hd_path) = home_dir() {
+        hd_path.push(".netrc");
```

Comment:
HTTPie/`requests` [looks at](https://github.com/psf/requests/blob/913880c45a3a8c3bf6b298e9c38709cd95a9c97c/requests/utils.py#L172) `$NETRC` first and considers both `~/.netrc` and `~/_netrc`, maybe we should do that too?

Tests shouldn't look at the real netrc, or some of them will fail if you have one. I guess setting a `$HOME` environment variable in [`get_base_command()`](https://github.com/ducaale/xh/blob/5624e8e1e0ba5da6d38398735e6da7de810da0d9/tests/cli.rs#L16) would do the trick. Tests that need a fake homedir can then override it once more with a temporary directory.